src/main/kotlin/org/wycliffeassociates/otter/common/domain/usfm/ParseUsfm.kt

```diff
@@ -0,0 +1,89 @@
+package org.wycliffeassociates.otter.common.domain.usfm
+
+import java.io.File
+
+val sep = System.lineSeparator()
+
+val MARKER_BOOK_NAME = "\\id"
+val MARKER_CHAPTER_NUMBER = "\\c"
+val MARKER_VERSE_NUMBER = "\\v"
+var MARKER_NEW_PARAGRAPH = "\\p"
+var MARKER_SECTION_HEADING = "\\s"
+var MARKER_SECTION_HEADING_ONE = "\\s1"
+var MARKER_SECTION_HEADING_TWO = "\\s2"
+var MARKER_SECTION_HEADING_THREE = "\\s3"
+var MARKER_SECTION_HEADING_FOUR = "\\s4"
+var MARKER_CHUNK = "\\s5"
+
+
+data class Current(var c: Int = 1, var v: Int = 1)
+data class Verse(val number: Int, var text: String)
```

Comment:
Should number also be `var`?

Reply:
no; the number doesn't change. Text may need more text appended to it because of how USFM works, there can be new lines that haven't moved to a new verse, but add styling to a section of text.